sklearn/preprocessing/_polynomial.py

```diff
@@ -1140,8 +1269,9 @@ def transform(self, X):
                     " transformer to produce fewer than 2^31 output features"
                 )
             XBS = sparse.hstack(output_list, format="csr")
+        # TODO: Remove conversion to csr, once scipy 1.10 is the minimum version:
         elif self.sparse_output:
```

Comment:
```suggestion
        elif self.sparse_output:
            # TODO: Remove conversion to csr, once scipy 1.10 is the minimum version:
```
This comment is not meant for the `hstack` command above but for `sparse.csr_matrix(XBS)`, right?